nvme.c

```diff
@@ -6573,14 +6575,18 @@ static int copy_cmd(int argc, char **argv, struct command *cmd, struct plugin *p
 	const char *d_format = "source range entry format";
 
 	_cleanup_nvme_dev_ struct nvme_dev *dev = NULL;
-	uint16_t nr, nb, ns, nrts, natms, nats;
+	uint16_t nr, nb, ns, nrts, natms, nats, nid;
 	__u16 nlbs[128] = { 0 };
 	__u64 slbas[128] = { 0 };
+	__u32 snsids[128] = { 0 };
+	__u16 sopts[128] = { 0 };
 	int err;
 
 	union {
 		__u32 f0[128];
 		__u64 f1[101];
+		__u32 f2[128];
+		__u64 f3[101];
```

Comment:
Can `f2` and `f3` reuse `f0` and `f1`? That would avoid some code duplication below.